packages/autocomplete-js/src/__tests__/autocomplete.test.ts

```diff
@@ -103,7 +103,7 @@ describe('autocomplete-js', () => {
                       type="rotate"
                       values="0 50 50;90 50 50;180 50 50;360 50 50"
                     />
-
+                    
```

Comment:
this line is probably not needed

Reply:
Not sure why these spaces keep appearing/disappearing in the snapshot.